master/pkg/model/experiment.go

```diff
@@ -61,6 +63,12 @@ const (
 	TrialWorkloadSequencerType WorkloadSequencerType = "TRIAL_WORKLOAD_SEQUENCER"
 )
 
+// StateFromProto maps experimentv1.State to State.
+func StateFromProto(state experimentv1.State) State {
```

Comment:
was this used? also i prefer an exhaustive switch with panic as opposed to parsing something into state that isn't a valid enum (e.g. if proto gets STATE_WEIRD_STATE, since state is just a string we can parse WEIRD_STATE into it, which is invalid)

Reply:
not here. and that makes sense it was just me being lazy..